### DISCUSSION TITLE
how do i delete a reply function

### DISCUSSION THREAD

**Discussion by Gregmanz** [victornpb/undiscord#413] [1. Questions & Answers]
how would I delete a reply function because the bot doesn't detect all mentions

**Comment by abigailwillow** [comment]


**Reply by Gregmanz** [reply]
> Could you please elaborate?

you know how when someone replies to you, it pings you. How do i remove those?

**Reply by Gregmanz** [reply]
When you look up your mentions it says everyone who has mentioned you but the ones who pinged you through reply function doesnt work

![image](https://user-images.githubusercontent.com/114528495/192633738-72b2580f-d113-4b4d-9f79-dbdc89fd7135.png)

**Reply by Gregmanz** [reply]
> As it says in the screenshot, the original message was deleted. That is the default behavior for Discord when someone has replied to a message you sent. Their message does not get deleted when you delete yours.

is there a way for me to delete them because it still shows up as mentions

**Reply by Gregmanz** [reply]
I want to delete their quote of what i said which is deleted

**Reply by Gregmanz** [reply]
> As far as I can tell, in the screenshot you provided, your message was deleted.

no like I want to delete their reply to my deleted message

**Reply by Gregmanz** [reply]
> As I mentioned, that is not possible if you don't have the permissions to do that in a guild, or if you're in a DM conversation.

I have permission in the server..

**Reply by Gregmanz** [reply]
i want to know how to do it automictically

**Reply by Gregmanz** [reply]
> I understand your issue now, however I don't think that's currently possible using this userscript. You can manually find all of them by searching for `mentions:id` where `id` is your user snowflake.

hm well i tried that ig thats the only way thanks

**Comment by last1melody** [comment]
I've been looking for it and it looks like no one has tried to make it, so I added this feature on my own by making changes to the code. It only works on Tampermonkey.
You still need permissions to delete these messages. Always.
[undiscord_with_mention_quotes.zip](https://github.com/victornpb/undiscord/files/14622533/undiscord_with_mention_quotes.zip)
![image](https://github.com/victornpb/undiscord/assets/116696935/ef065317-44a6-4a28-bf76-bcb798d3293e)
